setup.py

```diff
@@ -42,12 +50,7 @@
         'aiodocker',
     ],
     python_requires='>=3.5',
-    install_requires=[
-        'setuptools==36.2.0',
-        'aiohttp==2.2.3',
-        'yarl>=0.10',
-    ],
-    extras_require={
-        'test': ['pytest', 'pytest-asyncio', 'flake8'],
-    }
+    install_requires=requirements,
```

Comment:
Relax it to non-strict check, e.g. `aiohttp>=2.0` where `2.0` is minimal supported version.
Sorry, I don't follow what minimal aiohttp version is supported by aiodocker.